iFixit/react-commerce
# Migrate react-instantsearch to v7

This PR upgrade react-instantsearch dependencies from `react-instantsearch-hooks-*` to the new v7 `react-instantsearch-*` package naming convention.

### QA

1. Visit Vercel preview
2. Verify that pages including instantsearch functionalities (`/Parts`, `/Tools`, `/Store`...) are still working as they did on v6.
